### PR TITLE
Update docs to remove exclusion of Connected Content from Content Car…

### PR DESCRIPTION
**Pull request by christorbraze**
…ds and clarify sending

# Pull Request/Issue Resolution

**Description of Change:**
Update docs to remove comment saying that Connected Content is not supported in Content Cards. Similarly, adjust note related to retries to be consistent with this (retries are only unsupported for IAM).

Also add a note about when the Connected Content call is made, which is at send time for all message types with the exception of IAMs, which makes the call at view time.

**Reason for Change:**
Connected Content will be supported in Content Cards shortly.

Closes #**ISSUE_NUMBER_HERE**
PC-1013
https://jira.braze.com/browse/PC-1013

---
---

## PR Checklist
- [ ] Ensure you have completed our CLA.
- [ ] Tag @EmilyNecciai as a reviewer when the your work is done and ready to be reviewed for merge. 
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide) to be sure you're utilizing the proper markdown formatting.
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices) to be sure you're aligning with our voice and other style best practices.
- [ ] [Preview your deployed changes](https://homeslice.braze.com/docs) to confirm that none of your changes break production. Pay close attention to links and images.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

---
---

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/connected_content_retries.md [hunk 1]
```diff
@@ -7,22 +7,22 @@ page_order: 4
 
 # Connected Content Retries
 
-{% raw %}
-
 Because Connected Content relies on receiving data from APIs, there is the possibility that an API is intermittently unavailable while Braze makes the call. In this case, Braze supports retry logic to re-attempt the request using exponential backoff. To enable retries, add `:retry` in the Connected Content call, as shown below:
-
+{% raw %}
```
Comment (christorbraze) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/connected_content_retries.md]: Raw was only needed in this section that has other `{%` inside

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/connected_content_retries.md [hunk 2]
```diff
@@ -7,22 +7,22 @@ page_order: 4
...
+{% endraw %}
 
 If the API call fails and this is enabled, Braze will retry the call while respecting the [rate limit][47] you set for each resend. Braze will move any failed messages to the "back of the queue" and add additional minutes, if necessary, to the total minutes it would take to send your message.
 
 Please note that if a retried attempt succeeds, the message is sent and no further retries are attempted for that message. If the Connected Content call errors out 5 times, the message is aborted similar to if an [abort message tag][1] was triggered.
 
-Connected Content `:retry` is available in webhooks, emails, and push notifications.
+{% alert note %}
+Connected Content `:retry` is not available for In-App Messages.
+{% endalert %}
```
Comment (christorbraze) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/connected_content_retries.md]: This seemed to me important enough to embed in a note... but happy to remove that bit if it doesn't fit the kind of thing we want to put in to a `Note:` section.

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/making_an_api_call.md [hunk 3]
```diff
@@ -26,13 +26,16 @@ Hi, here are some articles that you might find interesting:
...
+{% alert note %}
+* Attribute values must be surrounded by `${}` in order to operate properly within Braze's version of Liquid Syntax.
+* Connected Content calls will happen at the time the message is sent, with the exception of In-App Messages, which will make this call at the time the message is viewed.
+* Connected Content calls do not follow redirects.
+* Braze's systems may make the same Connected Content API call more than once per recipient. That is because Braze may need to make a Connected Content API call to render a message payload, and message payloads can be rendered multiple times per recipient for the purposes of validation, retry logic, or other internal purposes. Your systems should be able to tolerate the same Connected Content call being made more than one time per recipient.
+{% endalert %}
```
Comment (christorbraze) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/making_an_api_call.md]: Previously these notes were represented as quotations...but they really seemed more like a list of notes, so I changed them here...again happy to change.

Also I added a second bullet here about the timing of the API calls wrt message sending and viewing.

**Comment by christorbraze** [comment]
Do I need to do the `Contributor License Agreement`?

**Comment by zzhaobraze** [comment]
@cla-bot check

**Comment by zzhaobraze** [comment]
@christorbraze @froodian @Bucimis  is this good to be merged for deploy next tuesday?

**Comment by Bucimis** [comment]
@zzhaobraze thanks for checking! not yet, the feature needs to pass CI first - we'll give the signal

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/making_an_api_call.md (hunk 3 above)
Comment (EmilyNecciai) [reply _docs/_user_guide/personalization_and_dynamic_content/connected_content/making_an_api_call.md]: I'll take a look and edit if necessary! Thank you! :)

**Comment by EmilyNecciai** [comment]
@Bucimis Should this go out tomorrow?

**Comment by christorbraze** [comment]
I think we'll probably hold off for at least one more week...Jared and PM both on vacation ;-)

**Comment by EmilyNecciai** [comment]
@christorbraze @Bucimis @froodian Is this good to go out today?

**Comment by christorbraze** [comment]
Oh, sorry I didn't see your mail...came here to comment anyway.

This is scheduled to go "live" on July 9, so probably push back until then. Thanks for keeping an eye on this!

https://confluence.braze.com/display/PM/Release+on+2019-07-09

**Pull request merged**